### ISSUE TITLE
Missing annotations.proto on Windows 10

### ISSUE THREAD

**Issue by jamaljsr** [RadarTech/lnrpc#59]
This issue is related to #56. 

I've upgraded my app to use `v0.8.0-beta.3`. Everything works great on OSX, but on Windows, there is still an issue loading the google proto files. I get this error when calling `createLnRpc`:
```
ENOENT: no such file or directory, open 'C:\\...\\node_modules\\@radar\\lnrpc\\lnd\\v0.8.0-beta\\google\\api\\annotations.proto'
```
I went digging into my node_modules and noticed the google dir is missing.There's only the `rpc.proto` file there. I went ahead and copied the google dir from the root lnrpc dir. 
```
$ tree lnd/
lnd/
└── v0.8.0-beta
    ├── google
    │   └── api
    │       ├── annotations.proto
    │       └── http.proto
    └── rpc.proto
```
Now when running my app, I get a slightly different error:
```
ENOENT: no such file or directory, open 'C:\\...\\node_modules\\@radar\\lnrpc\\lnd\\v0.8.0-beta\\google\\api\\google\\api\\http.proto'
```
Its looking for a google dir inside of the google dir. So I created a nested copy.
```
$ tree lnd/
lnd/
└── v0.8.0-beta
    ├── google
    │   └── api
    │       ├── annotations.proto
    │       ├── google
    │       │   └── api
    │       │       ├── annotations.proto
    │       │       └── http.proto
    │       └── http.proto
    └── rpc.proto
```
Now when I run my app, I get another error:
```
ENOENT: no such file or directory, open 'C:\\...\\node_modules\\@radar\\lnrpc\\lnd\\v0.8.0-beta\\google\\api\\google\\protobuf\\descriptor.proto'
```
Its looking for a descriptor.proto file now. I was able to find it [here](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.proto). I saved it into the path, so now I have:
```
$ tree lnd/
lnd/
└── v0.8.0-beta
    ├── google
    │   └── api
    │       ├── annotations.proto
    │       ├── google
    │       │   ├── api
    │       │   │   ├── annotations.proto
    │       │   │   └── http.proto
    │       │   └── protobuf
    │       │       └── descriptor.proto
    │       └── http.proto
    └── rpc.proto
```
Finally, my app is able to connect to the LND node.

Just to be certain this wasn't being caused by a configuration issue in my app, I created a simple [nodejs app](https://github.com/jamaljsr/lnrpc-test). It produces the same error message about missing annotations.proto.

**Comment by cavanmflynn** [comment]
Thanks, @jamaljsr. Looking into this issue.

**Comment by cavanmflynn** [comment]
A fix has been published in [v0.8.0-beta.4](https://www.npmjs.com/package/@radar/lnrpc/v/0.8.0-beta.4).

**Comment by jamaljsr** [comment]
@cavanmflynn wow, that was fast! i’ll give it a try. Thanks
